### ISSUE TITLE
Missing alt text in the Collaborative GitHub Workflow image

### ISSUE THREAD

**Issue by infinite0x20** [pyOpenSci/lessons#134]
**Context:** [Contribute Open Source Lessons](https://www.pyopensci.org/lessons/contribute-open-source/your-first-contribution.html#an-overview-of-the-collaborative-github-workflow)

**Issue:** I noticed that the image describing the workflow doesn't have alt text. 

<img width="627" height="334" alt="Image of the collaborative GitHub workflow source HTML, which has alt text reading, 'alt text here'" src="https://github.com/user-attachments/assets/dc366a7f-44f2-46a7-a169-b938ee99cfd9" />

It'd be great to add an alt text description to improve web accessibility!

**Comment by infinite0x20** [comment]
(spoke with Leah at SciPy and will be working on this issue!)
